sechub-scan/src/test/java/com/daimler/sechub/domain/scan/HtmlCodeScanDescriptionSupportTest.java

```diff
@@ -0,0 +1,63 @@
+package com.daimler.sechub.domain.scan;
```

Comment:
Please add everywhere the `/*prepare*/, /* execute*/` and `/*test*/` parts.

Reply:
done